src/layout/regions.macros.html

```diff
@@ -181,16 +181,19 @@ <h4 class="footer-nav-group-title -light">Read More</h4>
         {{ icon.svg("logo")}}
         <span class="nav-item-text is-invisible">Home</span>
       </a>
-      <ul class="nav-main-group nav-main-group--user">
-        <li role="presentation">
-          <span class="phone-number js-a-sup-phone">
-            <a class="nav-item" data-track-click-target="phone" href="tel:1{{ phone_number|replace('-', '') }}">
-              {{ icon.icon("phone")}}
-              {{ phone_number }}
-            </a>
-          </span>
-        </li>
-      </ul>
+
+      {% if phone_number %}
+        <ul class="nav-main-group nav-main-group--user">
+          <li role="presentation">
```

Comment:
To be honest, I don't think this needs a complete ul structure just for the phone number. We used role presentation here because in this context list isn't really used for its semantic meaning. 

Shave some bytes and just get rid of the ul structure all together